rfc-013-code-repositories.md

```diff
@@ -0,0 +1,77 @@
+# Code repository standards
+
+## Summary
+
+A set of standards to be used for projects that will be shared within Made Tech or the greater technology community. These are projects hosted on GitHub, Bitbucket, Gitlab or similar public source control management.
+
+## Problem
+
+Made Tech code repositories (to be referred to as repos herein) do not have a common look and feel.
+
+## Visibility
+
+The visibility of your project **SHOULD** mean the following:
+
+- You **MUST** set your repo to **Private to creator**  if code isn’t ready for wide consumption.
+- You **MUST** set your repo to **Private to the organisation** when the code is propriety or NDA bound.
+- You **SHOULD** set your repo to **Public**  if it can be used by the community at large. It **MUST** also come with `README.md` and `LICENSE`
+
+## Files in the Repository
+
+### README.md
+
+It **MUST** exist for **Private to the organisation** and **Public**.
```

Comment:
Not sure what you mean here - assume that README should always be present :smile:

Reply:
Yes that's it, but I will update to make it clearer. Something like this:

> The `README.md` **MUST** be present for **Private to the organisation** and **Public**.

Reply:
| The README.md MUST be present. 

? 

I.e. doesn't matter how public or private it is pls readme it